randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -806,6 +808,7 @@ Extra - asset_id: collision_camera_012
               Gravity Suit
               Flash Shift or Simple IBJ or Use Spin Boost
           Grapple Beam and Grapple Movement (Beginner) and Use Spin Boost
+          # https://www.youtube.com/watch?v=tyQFmi8slA4
           Grapple Beam and Grapple Movement (Intermediate)
```

Comment:
Can probably add in my video with Spin Boost for that one. Up to you which one to use for jumpless.
https://www.youtube.com/watch?v=nDV0bR4sSMA